xbmc/windowing/X11/WinSystemX11GLContext.cpp

```diff
@@ -261,25 +261,36 @@ bool CWinSystemX11GLContext::RefreshGLContext(bool force)
   VIDEOPLAYER::CRendererFactory::ClearRenderer();
   CLinuxRendererGL::Register();
 
-  m_pGLContext = new CGLContextEGL(m_dpy);
-  success = m_pGLContext->Refresh(force, m_nScreen, m_glWindow, m_newGlContext);
-  if (success)
+  std::string gpuvendor;
+  const char* vend = (const char*) glGetString(GL_VENDOR);
+  if (vend)
+    gpuvendor = vend;
+  std::transform(gpuvendor.begin(), gpuvendor.end(), gpuvendor.begin(), ::tolower);
+  bool isNvidia = (gpuvendor.compare(0, 6, "nvidia") == 0);
+  bool isIntel = (gpuvendor.compare(0, 5, "intel") == 0);
+  std::string gli = getenv("GL_INTERFACE");
```

Comment:
who sets this env var?

Reply:
devs or users who want to force a particular environment

Reply:
Wouldn't it be better to call it `KODI_GL_INTERFACE` then? `GL_INTERFACE` makes it look like it's a mesa (or similar) env variable.

Reply:
Makes sense. But then we should change others like AE_SINK too

Reply:
Definitely. But I think it's most important for the newly introduced ones since nobody is supposed to be using it yet so we don't have to worry about compatibility.

Reply:
compatibilty is not a requirement we have to consider in this phase. we are still changing APIs for python and json-rpc. that is by far more critical than a few environment variables, which are not used by the average joe